test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoMockTest.java

```diff
@@ -131,7 +131,7 @@ public void testGetActivities() throws Exception {
         ScheduleContext context = new ScheduleContext.Builder()
             .withStudyIdentifier(TEST_STUDY)
             .withClientInfo(ClientInfo.UNKNOWN_CLIENT)
-            .withInitialTimeZone(PACIFIC_TIME_ZONE)
+            .withRequestTimeZone(PACIFIC_TIME_ZONE)
```

Comment:
There are a lot of tests where we're replacing initialTimeZone with requestTimeZone instead of endsOn.getTimeZone() with requestTimeZone. This changes the semantics of a lot of tests, and means we could be losing test coverage. Are we sure we want to do this?